CSA.md

```diff
@@ -1,91 +1,93 @@
 
-      (a) Except as expressly permitted by this Agreement, <span class="coverpage_link">Customer</span> will not (and will not allow any anyone else to): (i) reverse engineer, decompile, or attempt to discover any source code or underlying ideas or algorithms of the Cloud Service (except to the extent Applicable Laws prohibit this restriction); (ii) provide, sell, transfer, sublicense, lend, distribute, rent, or otherwise allow others to access or use the Cloud Service; (iii) remove any proprietary notices or labels; (iv) copy, modify, or create derivative works of the Cloud Service; (v) conduct security or vulnerability tests on, interfere with the operation of, or circumvent access restrictions of the Cloud Service; (vi) access accounts, information, data, or portions of the Cloud Service to which <span class="coverpage_link">Customer</span> does not have explicit authorization; (vii) use the Cloud Service to develop a competing product; or (viii) use the Cloud Service with any High Risk Activities or with activity prohibited by Applicable Laws.
+2. ## Restrictions & Obligations  
 
-      (b) <span class="coverpage_link">Customer's</span> use of the Cloud Service must comply with the <span class="coverpage_link">Acceptable Use Policy</span>, if any.
+    2.1 <span class="header_3">Restrictions On Customer.</span>
 
-    2.2 <span class="header_3">Support.</span>  During the <span class="orderform_link">Subscription Period</span>, <span class="coverpage_link">Provider</span> will provide <span class="orderform_link">Product Support</span> as described in the Order Form, if any.  
+      (a) Except as expressly permitted by this Agreement, <span class="coverpage_link">Customer</span> will not (and will not allow any anyone else to): (i) reverse engineer, decompile, or attempt to discover any source code or underlying ideas or algorithms of the Product (except to the extent Applicable Laws prohibit this restriction); (ii) provide, sell, transfer, sublicense, lend, distribute, rent, or otherwise allow others to access or use the Product; (iii) remove any proprietary notices or labels; (iv) copy, modify, or create derivative works of the Product; (v) conduct security or vulnerability tests on, interfere with the operation of, cause performance degradation of, or circumvent access restrictions of the Product; (vi) access accounts, information, data, or portions of the Product to which <span class="coverpage_link">Customer</span> does not have explicit authorization; (vii) use the Product to develop a competing service or product; (viii) use the Product with any High Risk Activities or with activity prohibited by Applicable Lawss; (ix) use the Product to obtain unauthorized access to anyone else’s networks or equipment; or (x) upload, submit, or otherwise make available to the Product any Customer Content to which <span class="coverpage_link">Customer</span> and Users do not have the proper rights
```

Comment:
There was an extra "s" in Laws

```suggestion
      (a) Except as expressly permitted by this Agreement, <span class="coverpage_link">Customer</span> will not (and will not allow any anyone else to): (i) reverse engineer, decompile, or attempt to discover any source code or underlying ideas or algorithms of the Product (except to the extent Applicable Laws prohibit this restriction); (ii) provide, sell, transfer, sublicense, lend, distribute, rent, or otherwise allow others to access or use the Product; (iii) remove any proprietary notices or labels; (iv) copy, modify, or create derivative works of the Product; (v) conduct security or vulnerability tests on, interfere with the operation of, cause performance degradation of, or circumvent access restrictions of the Product; (vi) access accounts, information, data, or portions of the Product to which <span class="coverpage_link">Customer</span> does not have explicit authorization; (vii) use the Product to develop a competing service or product; (viii) use the Product with any High Risk Activities or with activity prohibited by Applicable Laws; (ix) use the Product to obtain unauthorized access to anyone else’s networks or equipment; or (x) upload, submit, or otherwise make available to the Product any Customer Content to which <span class="coverpage_link">Customer</span> and Users do not have the proper rights
```